src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubConfiguration.cs

```diff
@@ -278,6 +278,7 @@ void IExtensionConfigProvider.Initialize(ExtensionConfigContext context)
             cm.AddConverter<string, EventData>(ConvertString2EventData);
             cm.AddConverter<EventData, string>(ConvertEventData2String);
             cm.AddConverter<byte[], EventData>(ConvertBytes2EventData); // direct, handles non-string representations
+            cm.AddConverter<EventData, byte[]>(ConvertEventData2Bytes); // direct, handles non-string representations
```

Comment:
Just to make sure - we'll now handle both byte[] and byte[][] automatically, w/o the user having to register any other converters?